### PR TITLE
Uses React 16’s createPortal

### LINKED ISSUE #68: Update to make use of all React v16 goodness
This will constitute a breaking of the `react >= v0.14.0` peer dependency. Thus, I'd guess that `v2` and any minor or patch versions will be the last to support `< react v16`, and `v3` and above will support `react >= v16`.

https://facebook.github.io/react/blog/2017/09/26/react-v16.0.html
https://facebook.github.io/react/docs/portals.html

### PR DESCRIPTION
**Pull request by cbothner**
So, I needed this :joy:

In React 16, `ReactDOM.render` returns `null` in preparation for async rendering, so it never unzoomed after I upgraded.

Let me know if there’re any style/implementation changes you’d have me make, or of course if you see any bugs I missed.

Resolves #68

### CODE REVIEW

**Comment by rpearce** [comment]
I can't wait to review this. Need some time though

**Comment by rpearce** [comment]
Can you do a major version bump commit in here for me?

**Comment by rpearce** [comment]
I reckon we should also update the readme to explicitly say that for compatibility for react < v16, you need to use version `2.x`; for > v16, >= `3.x`

**Comment by cbothner** [comment]
I put the readme changes in with the version bump because it seemed right, but I don't have much experience publishing packages so I'm happy to change that if needed.

File: src/EventsWrapper.js [hunk 1]
```diff
@@ -35,23 +38,31 @@ export default class EventsWrapper extends Component {
...
 
-    return this.refs.child.unzoom()
+    return this.child.unzoom()
   }
 
   _cloneChild() {
     return React.cloneElement(React.Children.only(this.props.children), {
-      ref: 'child'
+      ref: child => {
+        this.child = child
```
Comment (rpearce) [inline src/EventsWrapper.js]: :+1:

**Comment by rpearce** [comment]
This all looks _really_ good. No issues from what I can see! Well done

**Pull request merged**